package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "disqus-react",
-  "version": "1.0.8",
+  "version": "1.0.9",
```

Comment:
We're going to want to leave this unchanged as it is updated automatically by the script we use to publish new package versions to npm.